### ISSUE TITLE
Ensure that we are routing Camp LeJeune cases correctly

### ISSUE THREAD

**Issue by lakohl** [department-of-veterans-affairs/caseflow#2939]
We should not be routing to RO 70 as it is just used as a place to store and keep track of Contaminated Water at Camp LeJeune appeals in VACOLS.

Instead, we should be routing them to RO 327, as they only have one RO code in VBMS.

Backstory and more context here: https://github.com/department-of-veterans-affairs/appeals-design-research/issues/362#issuecomment-321002536

**Comment by kierachell** [comment]
There is no RO 327. There is RO27 which maps to station 327. Which is where those cases go already. What we do have is cases that come from RO70. Should they get CWCL marked automatically ?

**Comment by lakohl** [comment]
@kierachell we have cases that come from RO70 in Dispatch?
What is CWCL?

**Comment by kierachell** [comment]
So in VACOLS an appeal will have Regional Office 70 (BFREGOFF column). It will come to us and depending on the Special Issue Category selected in Dispatch - the End Product created for that Appeal will be sent to a Station of Jurisdiction corresponding to that Regional Office.

Previously, we didn't have a Station of Jurisdiction for Regional Office 70. So it was causing errors in Dispatch. After some research I found that RO27 is in Louisville and so is RO70. RO27 is currently used for Special Issue Category- Contaminated Water Camp Lejeune (CWCL). So, presumably - Station 327 would work for both Regional Office 27 and 70. Right?

After some more research I found that RO70 is marked not just as a Louisville RO, but specifically Louisville CWCL RO. Why? I don't know. So, my question is: is it safe to assume that Appeals that have BFREGOFF = RO70 can be marked with Special Issue Category - CWCL automatically?

**Comment by lakohl** [comment]
@kierachell this sound right to me. @Chingujo and @abbyraskinUSDS to confirm.

**Comment by lakohl** [comment]
sounds like this is taken care of

**Comment by Chingujo** [comment]
Sorry for missing this but yes that sounds right. RO 327 owns both 70 and 24. For camp L they should be routed to RO 324. 70 is just a location they use to dump appeals but in terms of VBMS Louisville only has one code which is 327.

**Comment by kierachell** [comment]
### So uh.... nobody answered this:
> So, my question is: is it safe to assume that Appeals that have BFREGOFF = RO70 can be marked with Special Issue Category - CLCW automatically?

Kind of like we do for Private Attorney cases?
@cmgiven @lakohl @Chingujo ... anyone?

**Comment by Chingujo** [comment]
TLDR: Normally yes. However, there could be routing changes in the mix with Camp L according to Louisville coach. Less we have a way of receiving routing changes from AMO, might want to avoid auto checking until we find out what and when. 

Normally yes...as there is only one VACOLS code with Camp L and according to the coach at Louisville there is one location for these type of appeals and only one VBMS Station that deals with it (Station 372). 

However, Mike told me that there could be potential routing changes as there are conversations with AMO about how Camp L might change. Since we don't know when that will happen and at least to my knowledge, I don't know of any streams of communication where AMO updates us about routing changes, we probably should not. Less routing gets suddenly changes and we get blindsided.

**Comment by kierachell** [comment]
**PASSED**
just have to mark it
